addons/account/models/account_tax.py

```diff
@@ -990,16 +990,18 @@ def add_tax_values(record, results, grouping_key, serialized_grouping_key, tax_v
                     grouping_key = frozendict(self._get_generation_dict_from_base_line(base_line, tax_values))
 
                     total_amounts = amount_per_tax_repartition_line_id[grouping_key]
-                    tax_amount_currency_with_delta = tax_values['tax_amount_currency'] \
-                                                     + total_amounts['delta_tax_amount_currency']
-                    tax_amount_currency = currency.round(tax_amount_currency_with_delta)
-                    tax_amount_with_delta = tax_values['tax_amount'] \
-                                            + total_amounts['delta_tax_amount']
-                    tax_amount = comp_currency.round(tax_amount_with_delta)
-                    tax_values['tax_amount_currency'] = tax_amount_currency
-                    tax_values['tax_amount'] = tax_amount
-                    total_amounts['delta_tax_amount_currency'] = tax_amount_currency_with_delta - tax_amount_currency
-                    total_amounts['delta_tax_amount'] = tax_amount_with_delta - tax_amount
+                    tax_amount_currency = tax_values['tax_amount_currency']
```

Comment:
I removed the diff, the test is still passing.
Tested on the UI, a SO:
![image](https://github.com/odoo/odoo/assets/21679159/e894a68d-40ce-4855-a0ad-8590481aa75a)

and the invoice:
![image](https://github.com/odoo/odoo/assets/21679159/ea59ea10-fb57-44ea-a9cd-31e39a0bec09)

Where is the issue exactly?